packages/office-ui-fabric-react/src/components/Button/examples/Button.Action.Example.tsx

```diff
@@ -12,8 +18,11 @@ export class ButtonActionExample extends React.Component<IButtonProps> {
   public render() {
     let { disabled, checked } = this.props;
 
+    const getClassNames = classNamesFunction<IButtonBasicExampleStyleProps, IButtonBasicExampleStyles>();
+    const classNames = getClassNames(getStyles);
+
     return (
-      <div className='ms-BasicButtonsExample'>
+      <div className={ css(classNames.example) }>
```

Comment:
We don't need the css function for one class and at all anymore because merge styles takes care of merging a bunch of class names now.